src/CodeGen_LLVM.cpp

```diff
@@ -904,6 +904,8 @@ llvm::Type *CodeGen_LLVM::llvm_type_of(Type t) {
     return Internal::llvm_type_of(context, t);
 }
 
+llvm::TargetMachine *get_target_machine(const llvm::Module &module);
```

Comment:
Maybe all of this stuff (get_md_*, clone_target_options, get_target_machine, etc.) all belongs in CodeGen_Internal.h/cpp? It's a bunch of basic LLVM helper stuff.